qiita_db/study.py

```diff
@@ -206,6 +207,26 @@ def get_by_status(cls, status):
 
         return studies
 
+    @staticmethod
+    def get_by_portal(portal):
```

Comment:
I don't think that is the correct way of doing it. Also, performance wise, is going to kill us. That is a big no-no-no from my end.